packages/kotti-ui/source/kotti-filter/components/FilterRow.vue

```diff
@@ -0,0 +1,230 @@
+<template>
+	<KtForm
+		class="kt-filter-row"
+		hideValidation
+		:isLoading="isLoading"
+		:size="Kotti.Field.Size.SMALL"
+		:value="value"
+		@input="handleSetValue"
+	>
+		<span class="kt-filter-row__prefix" v-text="prefix" />
+		<KtFieldSingleSelect
+			class="kt-filter-row__column-select"
+			formKey="key"
+			hideClear
+			:options="columnOptions"
+		/>
+		<KtFieldSingleSelect
+			class="kt-filter-row__operation-select"
+			formKey="operation"
+			hideClear
+			:isDisabled="isOperationSelectDisabled"
+			:options="operationOptions"
+		/>
+		<div :class="valueContainerClasses">
+			<component
+				:is="valueComponent"
+				:collapseTagsAfter="1"
+				formKey="value"
+				:options="valueOptions"
+				type="switch"
+			/>
+			<span v-if="showValueState" v-text="valueState" />
+		</div>
+		<ButtonLink
+			class="kt-filter-row__remove"
+			:icon="Yoco.Icon.CLOSE"
+			:isLoading="isLoading"
+			:type="Kotti.Filter.ButtonLinkType.DANGER"
+			@click="handleRemove"
+		/>
+	</KtForm>
+</template>
+
+<script lang="ts">
+import { Yoco } from '@3yourmind/yoco'
+import { computed, defineComponent } from '@vue/composition-api'
+
+import {
+	KtFieldDateRange,
+	KtFieldMultiSelect,
+	KtFieldNumber,
+	KtFieldSingleSelect,
+	KtFieldText,
+	KtFieldToggle,
+	KtForm,
+} from '../../'
+import { useTranslationNamespace } from '../../kotti-translation/hooks'
+import { Kotti } from '../../types'
+import { getOperationOptions, getValueComponent } from '../utils'
+
+import ButtonLink from './ButtonLink.vue'
+
+export default defineComponent<{
+	column: Kotti.Filter.Column
+	columnOptions: Kotti.FieldSingleSelect.Props['options']
+	isFirstItem: boolean
+	isLoading: boolean
+	value: Kotti.Filter.Filter
+}>({
+	name: 'FilterRow',
+	components: {
+		ButtonLink,
+		KtFieldDateRange,
+		KtFieldMultiSelect,
+		KtFieldNumber,
+		KtFieldSingleSelect,
+		KtFieldText,
+		KtFieldToggle,
+		KtForm,
+	},
+	props: {
+		column: {
+			default: () => ({}),
+			type: Object,
+		},
+		columnOptions: {
+			default: () => [],
+			type: Array,
+		},
+		isFirstItem: {
+			default: false,
+			type: Boolean,
+		},
+		isLoading: {
+			default: false,
+			type: Boolean,
+		},
+		value: {
+			default: () => ({
```

Comment:
What’s the use-case for a default value?

Reply:
why not `null`, instead of initializing the attributes 

but in theory `:value` should never need a `default`